smartsim/_core/control/controller.py

```diff
@@ -69,23 +70,19 @@ def start(self, manifest, block=True):
         The controller will start the job-manager thread upon
         execution of all jobs.
         """
-        try:
-            self._launch(manifest)
-
-            # start the job manager thread if not already started
-            if not self._jobs.actively_monitoring:
-                self._jobs.start()
+        self._jobs.kill_on_interrupt = kill_on_interrupt
+        signal.signal(signal.SIGINT, self._jobs.signal_interrupt)
```

Comment:
I would add a brief comment like `registering custom signal handler for ^C (SIGINT)` to make it clearer and stand out a bit more